class Excp5 
{
	public static void main(String[] args) 
	{
		System.out.println("Start");
		try
		{
			System.out.println(10/0);
			System.out.println(10/2); // It will not get Executed
		}
		catch (ArithmeticException ae)
		{
			System.out.println("Inside Catch");
		}
		System.out.println("Stop");
	}
}
